CI/templates/pipeline_run.js

```diff
@@ -0,0 +1,126 @@
+var azdev = require("azure-devops-node-api")
+var atob = require('atob');
+var configurations = require('./configurations.json');
+
+if(configurations.testPipelineArgs == null) {
+    throw new Error("Missing testPipelineArg from configurations");
+}
+
+if(process.env.npm_config_jsonStringBase64 == null) {
+    throw new Error("Missing JSON string with appIDs and platforms");
+}
+
+if(process.env.npm_config_deviceAndroid == null) {
+    throw new Error("Missing Android version to use");
+}
+
+if(process.env.npm_config_deviceIos == null) {
+    throw new Error("Missing iOS version to use");
+}
+
+if(process.env.npm_config_pipelineID == null) {
+    throw new Error("Missing Android Pipeline ID");
```

Comment:
i guess we should chnage error message here. Missing mobile-ui-tests pipeline ID

Reply:
yeah will change on the template.